hedera-node/data/config/api-permission.properties

```diff
@@ -0,0 +1,68 @@
+# Crypto
```

Comment:
I think this file is git-ignored. Is there any particular reason that it is still committed anyway?